apps/envited-x-data-space/common/validator/shacl/shacl.constants.ts

```diff
@@ -1,19 +1,24 @@
 import { Schema } from './shacl.types'
 
 export const SCHEMA_MAP = {
-  [Schema.default]: '/shaclSchema.ttl',
+  [Schema.automotiveSimulator]: '/schemas/automotive-simulator_shacl.ttl',
   [Schema.environmentModel]: '/schemas/environment_shacl.ttl',
   [Schema.general]: '/schemas/general_shacl.ttl',
   [Schema.georeference]: '/schemas/georeference_shacl.ttl',
   [Schema.gx]: '/schemas/gx_shacl.ttl',
   [Schema.hdmap]: '/schemas/hdmap_shacl.ttl',
+  [Schema.leakageTest]: '/schemas/leaking-test_shacl.ttl',
   [Schema.manifest]: '/schemas/manifest_shacl.ttl',
+  [Schema.marketplaceInfo]: '/schemas/marketplace-info_shacl.ttl',
   [Schema.ositrace]: '/schemas/ositrace_shacl.ttl',
   [Schema.scenario]: '/schemas/scenario_shacl.ttl',
   [Schema.surfaceModel]: '/schemas/surface-model_shacl.ttl',
-  [Schema.domainMetadata]: '/schemas/domainMetadata_shacl.ttl',
-  [Schema.gxMetadata]: '/schemas/gxMetadata_shacl.ttl',
-  [Schema.marketplaceInfo]: '/schemas/marketplace-info_shacl.ttl',
+  [Schema.service]: '/schemas/service_shacl.ttl',
+  [Schema.simulatedModel]: '/schemas/simulated-model_shacl.ttl',
+  [Schema.simulatedSensor]: '/schemas/simulated-sensor_shacl.ttl',
+  // [Schema.surveyResultDataOferring]: '/schemas/survey-result-data-offering_shacl.ttl',
+  // [Schema.surveyServiceOferring]: '/schemas/survery-service-offering_shacl.ttl',
```

Comment:
@jdsika I'm not sure how to use this SHACL templates the name that are used for this is "survey" for both so it looks like they are not unique.